Logos/Darktrace.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90.79 99.09">
```

Comment:
The logo is not clear, since it does not have colors, only orange color dot is emphasized. Can you confirm the output (look and feel) of the logo? 
![DarkTrace](https://user-images.githubusercontent.com/73506651/100819821-d1899b00-3401-11eb-9d25-97a2fea727a3.PNG)

Reply:
Can you please verify the attached file and confirm if it is as expected. If it is, please update the logo here.
[Darktrace-New.zip](https://github.com/Azure/Azure-Sentinel/files/5646216/Darktrace-New.zip)

Reply:
All we ask is to change  fill="#fff"  ==>   fill="#2f3a56"
Because the logo background is white and it is not visible.

Reply:
@SunRift, could you help to verify the proposed logo. We want to start the deployment process on our side ASAP.

Reply:
Yes that looks great, I've updated my PR with the new .svg